nodedb.go

```diff
@@ -642,7 +642,10 @@ func (ndb *nodeDB) DeleteVersionsRange(fromVersion, toVersion int64) error {
 	}
 	ndb.cacheOrphansCancel = cancel
 	// clear the cached orphans
-	ndb.cachedOrphans = sync.Map{}
+	ndb.cachedOrphans.Range(func(key, value interface{}) bool {
+		ndb.cachedOrphans.Delete(key)
```

Comment:
Need to test this out, range delete might be dangerous, you can collect the keys to delete first and then delete them in another loop.

Reply:
Fixed.